packages/ui/package.json

```
@@ -4,6 +4,7 @@
   "license": "GPL-3.0-only",
   "scripts": {
     "build": "node --max_old_space_size=4096 ./node_modules/.bin/webpack --mode production --progress",
+    "queries-parse": "ts-node dev/scripts/parseSchema.ts",
     "queries:generate": "graphql-codegen --config codegen.config.yml && yarn lint:fix",
```

```suggestion
    "queries:patch": "ts-node dev/scripts/parseSchema.ts",
    "queries:generate": "graphql-codegen --config codegen.config.yml && yarn lint:fix",
    "queries": "yarn queries:patch && yarn queries:generate"
```

you can do something like the above